frontend/src/scenes/notebooks/Nodes/NotebookNodeQuery.tsx

```diff
@@ -206,7 +206,7 @@ export const NotebookNodeQuery = createPostHogWidgetNode<NotebookNodeQueryAttrib
     Component,
     heightEstimate: 500,
     minHeight: 200,
-    resizeable: (attrs) => attrs.query.kind === NodeKind.DataTableNode,
+    resizeable: true,
```

Comment:
The change that kicked off this whole PR!

Reply:
For another PR but I think that was the only place we have more than a primitive value. We should remove the ability to set resizable based on the props